src/test/java/uk/gov/hmcts/reform/pcs/ApplicationTest.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.pcs;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+class ApplicationTest {
+
+    @Test
+    void shouldInvokeSpringApplicationRun() {
```

Comment:
Never seen anyone test this :) please remove it. It should be ignored in the code coverage anyway